ganga/GangaCore/Lib/Interactive/InteractiveScriptTemplate.py.template

```diff
@@ -17,6 +17,17 @@ import time
 sys.path.insert( 0, '###GANGA_PYTHONPATH###' )
 
 statfileName = os.path.join( '###OUTPUTDIR###', '__jobstatus__' )
+logfile = os.path.join('###OUTPUTDIR###', 'interactive_log.txt')
+
+try:
+    statfile = open(statfileName, 'w')
```

Comment:
Why is the `statfile` opened here. It seems to be opened again a few lines later.